blazor/dropdown-list/style.md

```diff
@@ -201,6 +201,18 @@ The following code demonstrates how to display a tooltip when hovering over the
 
 ![Blazor DropDownList with tooltip](./images/style/blazor-dropdownlist-tooltip.png)
 
+### Tooltip using HTMLAttribute in dropdown component
+
+To display the tooltip in dropdown component, you need to add the title attribute through HtmlAttributes property, which update the attribute to the root element `input`
```

Comment:
To display the tooltip in the dropdown component, add the title attribute through the HtmlAttributes property, which updates the attribute to the root element `input.`